app/jobs/purge_soft_deleted_records_job.rb

```diff
@@ -121,6 +125,13 @@ def process_model(model, data_source:)
 
   def check_max_deleted(size)
     @total_deleted += size
-    throw :halt if @total_deleted >= @max_deleted
+    return unless @total_deleted >= @max_deleted
+
+    log("Reached maximum deletion limit of #{@max_deleted} records, halting job.")
+    throw :halt
+  end
+
+  def log(str)
+    @notifier.ping(str) # logs to cloudwatch
```

Comment:
Using notifier will send to slack and the rails log.  If you don't need it in slack, I'd probably just use `Rails.logger.info(str)`

Reply:
I don't see the `Rails.logger.info` logs appear in cloudwatch. I only see logs from notifier.. I can find an example to share from a job that logs both